bin/remove_empty_rows_and_columns.py

```diff
@@ -0,0 +1,107 @@
+"""
+Create a copy of CSV files in a directory with empty columns and rows removed.
+
+This is a program for use in development. It transforms source files by
+removing any columns with an empty heading and any empty rows before
+writing them to an output directory.
+"""
+
+import sys
+import glob
+import os
+import logging
+import csv
+from argparse import ArgumentParser
+
+VERSION = '1.2.1-alpha'
+
+
+def main():
+    """Create a copy of CSV files in a directory with empty columns and rows removed."""
+    parser = ArgumentParser(
+        description='Create a copy of CSV files in a directory with '
+                    'empty columns and rows removed.',
+        epilog=f'Version: {VERSION}')
+
+    parser.add_argument('-i', '--input-dir',
+                        type=str,
+                        required=True,
+                        help='Input directory containing CSV files to transform')
+
+    parser.add_argument('-o', '--output-dir',
+                        type=str,
+                        required=True,
+                        help='Output directory to write modified files')
+
+    args = parser.parse_args()
```

Comment:
Wondering what happens if someone sets input and output directories to the same (or aliases of each other)?

Reply:
Thinking maybe we should insist that the output directory is empty. If we don't then they could end up with a mix of files we processed and files that are already there.

Reply:
Should we enforce that it's totally empty? Or just empty of csv files? Might they conceivably want to have files of other types in there?

Reply:
Going with the `--force-overwrite` flag to allow the empty dir check to be skipped. Default is directory must be empty.